### PR TITLE
test: add a check about 'Autostart'  of the off VM

### PR DESCRIPTION
**Pull request by yunmingyang**

### CODE REVIEW

File: test/verify/check-machines [hunk 1]
```diff
@@ -129,6 +129,10 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
     def goToMainPage(self):
         self.browser.click(".machines-listing-breadcrumb li:first-of-type a")
 
+    def backToVmListFromVmPage(self):
```
Comment (KKoukiou) [inline test/verify/check-machines:132]: I already have a global function for this: 
`self.goToMainPage`

**Review by KKoukiou — CHANGES_REQUESTED** [review]
I saw you added `running` option in startVm function. I think running a function called `startVm` and keeping the VM shut off is a bit confusing. if you need this option I would rename if to createVm.

File: test/verify/check-machines (hunk 1 above)
Comment (yunmingyang) [reply test/verify/check-machines:132]: Thanks, will remove this method

**Comment by yunmingyang** [comment]
> I saw you added `running` option in startVm function. I think running a function called `startVm` and keeping the VM shut off is a bit confusing. if you need this option I would rename if to createVm.
Make sense. But this option is needed in some cases. And if creating another method, these two method  will be almost the same. So I think maybe changing 'startVm' to 'createVm' is the best way

**Comment by KKoukiou** [comment]
> > I saw you added `running` option in startVm function. I think running a function called `startVm` and keeping the VM shut off is a bit confusing. if you need this option I would rename if to createVm.
> 
> Make sense. But this option is needed in some cases. And if creating another method, these two method will be almost the same. So I think maybe changing 'startVm' to 'createVm' is the best way

Sure, feel free to rename it..

**Comment by yunmingyang** [comment]
> > > I saw you added `running` option in startVm function. I think running a function called `startVm` and keeping the VM shut off is a bit confusing. if you need this option I would rename if to createVm.
> > 
> > 
> > Make sense. But this option is needed in some cases. And if creating another method, these two method will be almost the same. So I think maybe changing 'startVm' to 'createVm' is the best way
> 
> Sure, feel free to rename it..

Renamed, hope that will not cause issues. :joy:

**Review by KKoukiou — APPROVED** [review]
thanks, retrying little bit the tests.

**Pull request merged**